workflows/prognostic_c48_run/runfv3

```diff
@@ -42,18 +42,7 @@ function createRun {
     fv3config="$2"
     runfile="$3"
 
-
-    if [[ $# != 3 ]]
-    then 
-        usage
-        exit 1
-    fi
-
-    verifyUrlEmpty "$url"
-
-    echo "Setting up iterations"
-    gsutil cp "$fv3config" "$url/fv3config.yml"
-    gsutil cp "$runfile" "$url/runfile.py"
+    create_run $url $fv3config $runfile
```

Comment:
if/when we refactor this script to python, I imagine we would define a python CLI `runfv3` which has subparsers `create`, `append` etc. That will be backwards compatible. For now, the bash script is still the public entrypoint and this `create_run` CLI is mean to be private.